src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/ArchiveAzureFileProcessor.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.juddata.camel.processor;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.ConsumerTemplate;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ArchiveAzureFileProcessor implements Processor {
+
+    @Value("${archival-file-names}")
+    List<String> archivalFileNames;
+
+    @Value("${active-blob-path}")
+    String activeBlobs;
+
+    @Value("${archival-cred}")
+    String archivalCred;
+
+    @Override
+    public void process(Exchange exchange) throws Exception {
+        Integer count = exchange.getProperty("CamelLoopIndex", Integer.class);
+        String date = new SimpleDateFormat("dd-MM-yyyy--HH-mm").format(new Date());
+        String fileName = archivalFileNames.get(count);
```

Comment:
keep the date format in application-yaml

Reply:
done